renovate.json

```diff
@@ -6,5 +6,15 @@
   "docker": {
     "enabled":  false
   },
-  "prConcurrentLimit": 10
+  "prConcurrentLimit": 10,
+  "packageRules": [
+    {
+      "matchPackageNames": ["node-fetch"],
+      "allowedVersions": "< 3.0.0"
+    },
+    {
+      "matchPackageNames": ["serialize-error"],
+      "allowedVersions": "< 9.0.0"
```

Comment:
Do you think we should use ESM instead?

Reply:
For now no, because it requires lots of change and testing in the repo. 

@praveenkumarct was investigating ESM vs CommonJS, AFAIK he said that he will create a backlog item to evaluate the migration efforts for this repo ?